src/routes/admin/Proposal.js

```diff
@@ -5,16 +5,44 @@ import { verifyAdmin } from "../../middlewares/checkAuth";
 const router = express.Router();
 
 router.get("/get", verifyAdmin, async (req, res) => {
-
+    const proposals = await Proposal.find({});
+    res.send(proposals);
 });
 
 router.get("/get/:id", verifyAdmin, async (req, res) => {
-	
+	const _id = req.params.id;
+    const proposal = await Proposal.findById({ _id });
+    res.send(proposal);
 });
 
-
-router.delete("/delete/:id", verifyAdmin, async (req, res) => {
-
+router.get('/newproposal', verifyAdmin, async (req, res) => {
+    res.send('New proposal input field');
+})
+
+router.post('/newproposal', verifyAdmin, (req, res) => {
```

Comment:
should admins be able to create new proposals?

Reply:
```suggestion
router.post('/newproposal', verifyClub, (req, res) => {
```

Reply:
this route goes in club/Proposal.js